updates/commands.go

```diff
@@ -33,6 +35,8 @@ func handleCommandUpdate(fatBotUpdate FatBotUpdate) error {
 		}
 	case "status":
 		msg = handleStatusCommand(update)
+	case "summary":
+		handleMonthlySummaryCommand(fatBotUpdate)
```

Comment:
The convention here should be like the other commands -> `msg` object is returned from the process and then sent to the user below. This may also save you from sending arguments down the pipeline of functions